zebra-state/src/on_disk.rs

```diff
@@ -0,0 +1,172 @@
+//! The primary implementation of the `zebra_state::Service` built upon an embedded database
```

Comment:
```suggestion
//! The primary implementation of the `zebra_state::Service` built upon sled
```

Reply:
also (tioli) should this be sled.rs to match SledState?

Reply:
I prefer `on_disk` because I think `sled` should stay an implementation detail as much as possible, and I think `on_disk` gets the point across well of how the service functions.